scripts/E2ETestDeploy.s.sol

```diff
@@ -82,11 +84,13 @@ contract E2ETestDeploy is Script, IICS07TendermintMsgs {
         );
 
         // Deploy IBC Eureka with proxy
-        ICS26Router ics26RouterLogic = new ICS26Router();
-        ICS20Transfer ics20TransferLogic = new ICS20Transfer();
+        address escrowLogic = address(new Escrow());
```

Comment:
We can add this later, but I want to add some kind of tests that verifies that all logic contracts are indeed disabled (just for regression)